plugins/exporter.koplugin/clip.lua

```diff
@@ -239,7 +247,10 @@ function MyClipping:parseHighlight(highlights, bookmarks, book)
             for _, bookmark in pairs(bookmarks) do
                 if bookmark.datetime == item.datetime and bookmark.text then
                     local tmp = string.gsub(bookmark.text, "Page %d+ ", "")
-                    clipping.text = string.gsub(tmp, " @ %d%d%d%d%-%d%d%-%d%d %d%d:%d%d:%d%d", "")
+                    local bookmark_quote = string.gsub(tmp, " @ %d%d%d%d%-%d%d%-%d%d %d%d:%d%d:%d%d", "")
+                    if bookmark_quote ~= clipping.text then
+                        clipping.note = bookmark.text
```

Comment:
I suggest `clipping.note = bookmark_quote` as we have page number and timestamp shown in other clipping fields already.